Added jq requirement to examples readme (#106)
swift-server/swift-aws-lambda-runtime: Examples/LambdaFunctions/README.md

how to write a simple Lambda function in Swift, and how to package and deploy it
to the AWS Lambda platform.

> Note: You must have the [jq](https://stedolan.github.io/jq/download/) command line tool installed.

Reviewer: ```suggestion
Note: This example scripts assume you have [jq](https://stedolan.github.io/jq/download/) command line tool installed.
```

  ```

  Note: This script assumes you have AWS CLI installed and credentials setup in `~/.aws/credentials`.
  

>   Note: You must also have the [jq](https://stedolan.github.io/jq/download/) command line tool installed.

Reviewer: I think this is also required by the SAM example below, so maybe move this ~L6?
